src/domain/action-item.ts

```diff
@@ -14,19 +20,34 @@ export abstract class BaseActionItem {
   abstract get type();
 }
 
+export class DeadLetterQueue extends BaseActionItem {
+
+  constructor(report: DeadLetterQueueReport) {
+    super();
+    this.created = Date.now();
+    this.priority = report.call_failed ? ACTION_PRIORITY_IGNORE : ACTION_PRIORITY_NOW;
+    this.url = ''; // TODO???
```

Comment:
:eyes:

Reply:
yeah i don't know what to do here.  there isn't really a good programmatic url to provide.  @MatthewMazaika you okay with that just being an empty string?  should I remove the todo?

Reply:
per our slack convo i expanded the comment with more info

Reply:
this is the URL you go to when you click a DLQ item? i take it each DLQ doesn't have a URL? or maybe the URL isn't easily buildable at this point in the code? 

do we have a help doc and/or README somewhere that explains DLQ and then we can link each item to that? or maybe a doc that explains what a dev is supposed to do when they have something on their DLQ? now that i type all that out, if that kind of doc doesn't exist, maybe it should?

Reply:
so I could send you to your service in the eng hub today, but you wouldn't be able to do any of the management stuff without a provided supportal url which too me screams configuration smell.

if eng sys had dlq management i oculd easily construct a url.  but there just isn't a good url to put here imo.  ideally you see this action item and immediately go to your dlq in your supportal.